src/librustdoc/html/static/sidebar-items.js

```diff
@@ -0,0 +1 @@
+initSidebarItems({});
```

Comment:
Ah ok I see. Sorry but it's unneeded here. Please remove this file and all the linked code.

Reply:
Why is it uneeded? There needs to be this call _somewhere_, it doesn't currently exist in any JS.

Reply:
Because we already have the crate list stored and it's already accessible. I fixed it in #84834 without needing any of this.